src/napari_imagej/widgets/napari_imagej.py

```diff
@@ -188,3 +217,26 @@ def _finalize_info_bar(self):
         self.widget.info_box.version_bar.setText(
             " ".join(["ImageJ", str(ij().getVersion())])
         )
+
+    def _finalize_exception_printer(self):
+        @JImplements(["org.scijava.event.EventSubscriber"], deferred=True)
+        class NapariEventSubscriber(object):
+            @JOverride
+            def onEvent(self, event):
+                log_debug(str(event))
+
+            @JOverride
+            def getEventClass(self):
+                return jc.SciJavaEvent.class_
+
+            @JOverride
+            def equals(self, other):
+                return isinstance(other, NapariEventSubscriber)
```

Comment:
Let's also make sure that this is tested.

Reply:
I'm testing registration! I don't know that we can test that it actually works - we'll just assume it works.